src/app/ascension-store-modal/ascension-store-modal.component.html

```diff
@@ -55,6 +55,6 @@
     </tr>
   </table>
   <span *ngIf="characterService.characterState.empowermentFactor > 1">
-    You have consumed {{(characterService.characterState.empowermentFactor - 1) * 100 | number:'1.0-0'}} Empowerment Pills which multiplies all your attribute gains by {{characterService.characterState.getEmpowermentMult()  | number:'1.0-3'}}. You will always keep this bonus, even after ascending.
+    You have consumed {{(characterService.characterState.empowermentFactor - 1) * 100 | number:'1.0-0'}} Empowerment Pills which multiplies all your attribute gains by {{characterService.characterState.getEmpowermentMult()  | number:'1.0-3'}} (except you cannot surpass human limits). You will always keep this bonus, even after ascending.
```

Comment:
Probably something more like *subject to human limitations

Reply:
I don't think that's clear enough; 100000 times 10 is still 1000000 unless it's clear that the number given is a lie. The usage of "except" is supposed to help clarify this.